browser.css

```diff
@@ -64,6 +64,15 @@ img.larrybird {
 /* remove bottom border from active navbar tab */
 #react-root ._1Qfdd7IP {
 	box-shadow: none !important;
+	color: #1da1f2 !important;
+}
+
```

Comment:
Can you please add a comment in here so we know why we need to add this? Thanks.